### ISSUE TITLE
Error failed to fill whole buffer?

### ISSUE THREAD

**Issue by Leon279** [trumank/uesave-rs#30]
So im using "uesave edit F7B72DF0000000000000000000000000.sav" and i get the following error: "Error: at offset 3916: io error: failed to fill whole buffer". Do you know why this error occurs or did i do something wrong? to-json doesn't work either

**Comment by AndyClausen** [comment]
Looks like a palworld player save, right? I'm getting a similar error for the `Level.sav` file, here is full:
```
$ uesave edit Level.sav
Found non-standard magic: [87, e1, 79, 0b] (��y
                                               ) expected: GVAS, continuing to parse...
Error: at offset 10496326: io error: failed to fill whole buffer
```

**Comment by mandar1jn** [comment]
Should be closed as seen in #22

**Comment by AndyClausen** [comment]
True, we should be using the other repo mentioned there.
